co30_Domination.Altis/server/fn_createdrop.sqf

```diff
@@ -82,7 +82,7 @@ _wp setWaypointForceBehaviour true;
 private _vecdist = _chopper distance2D _drop_pos;
 
 //_chopper flyInHeight 100;
-#define __dist_to_drop 150
+#define __dist_to_drop 300

```

Comment:
300 is better, 150m did not work well with fixed-wing airdrop vehicles (the flyover altitude is higher)